packages/tsx-editor/src/utilities/defaultSupportedPackages.ts

```diff
@@ -1,7 +1,7 @@
 import { IPackageGroup } from '../interfaces/index';
 
 const fabricGroup: IPackageGroup = {
-  globalName: 'Fabric',
+  globalName: 'FluentUIReact',
```

Comment:
I think there's a test you'll need to update as well. (search for Fabric in tsx-editor)